### PR TITLE
Creating404 route

### PR DESCRIPTION
**Pull request by ridwvn22**
### Changes
Create an Errors controller and route
Create an Errors show that displays the below styling

### Link to Issue 
https://app.asana.com/0/1206510424537042/1206941111081264/f

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 
No Comments

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!
Screenshot 2024-04-23 at 2 16 27 PM

### Before you submit for review, Did you...

- [X] Added relevant tests (if applicable)
- [X] Linked Asana ticket to PR
- [X] Added comments, notes, or questions for reviewer
- [X] Added relevant screenshots
- [X] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: app/assets/stylesheets/404.css
Comment (ridwvn22) [inline app/assets/stylesheets/404.css:1]: delete file

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+
```
Comment (ridwvn22) [inline app/controllers/application_controller.rb:4]: empty space

File: app/controllers/errors_controller.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  # rescue_from ActionController::RoutingError, with: :render_404
+
+  # def render_404
+  #   puts "Rendering 404 error page"
+  #   render template: 'errors/error', status: :not_found
+  # end
+
+  def show
+    @status_code = params[:id]
+    puts 'Ridwan'
```
Comment (ridwvn22) [inline app/controllers/errors_controller.rb:13]: remove puts, clean up this file

File: app/views/layouts/_404.html.erb
Comment (ridwvn22) [inline app/views/layouts/_404.html.erb:1]: delete not needed

File: config/application.rb
Comment (ridwvn22) [inline config/application.rb:1]: check if this works without it

File: config/environments/development.rb
Comment (ridwvn22) [inline config/environments/development.rb:1]: check if this works

File: db/.DS_Store
Comment (ridwvn22) [inline db/.DS_Store:1]: git ignore, clean up

File: config/environments/development.rb [hunk 3]
```diff
@@ -16,6 +16,8 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  config.public_file_server.enabled = true
```
Comment (edithemmings) [inline config/environments/development.rb:19]: Did this end up being needed? What happens if you remove it?

Comment (ridwvn22) [reply config/environments/development.rb:19]: It still functions correctly, I'll remove it ty.

**Comment by ridwvn22** [comment]
> It looks awesome!! Are you able to delete the .DS_Store files?

No, I could not find the .DS file

**Review by edithemmings — COMMENTED** [review]
Looks great! Good to go as soon as the unneeded config gets cleaned up. Don't worry about the .DS files, those can be cleaned up later

**Review by edithemmings — APPROVED** [review]
Looks great! Nice work!!

**Pull request merged**